pdr_backend/pdr_dashboard/callbacks/callbacks_feeds.py

```diff
@@ -185,7 +208,7 @@ def toggle_modal(selected_rows, is_open_input):
     )
     # pylint: disable=unused-argument
     def update_graphs(is_open, selected_rows, feeds_table_data):
-        content = ModalContent("feeds_modal", app.data)
+        content = ModalContent("feeds_modal", app.data, app.start_date)
```

Comment:
having app.start date on data will also remove the need for passing it around here